Rules Building.html

```diff
@@ -0,0 +1,131 @@
+                    At the beginning of the Build Phase, each player gets an allowance of new credits 
+                    depending on what turn it is and how well they did in the last round. The <b>Loser</b> 
+                    of the last round gets the full payout. Each player after that gets 5 fewer credits
+                    in the order of people who died.
+                </p>
+                <table>
+                    <thead>
+                        <th>Round</th>
+                        <th>1</th>
+                        <th>2</th>
+                        <th>3</th>
+                        <th>4</th>
+                        <th>5</th>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <th>Income:</th>
+                            <td>40</td>
+                            <td>80</td>
+                            <td>120</td>
+                            <td>125</td>
+                            <td>130</td>
+                        </tr>
+                    </tbody>
+                </table>
+                <p class="rules">
+                    At the start of round 1, everyone gets the full payout.
+                </p>
+            </div>
+            <div class="subsystem">
+                <p class="rules"><b>Buying:</b>
+                    To get the weapons that you'll need to blow up other player's Mechs, and to get the 
+                    subsystems that you'll need to keep from being blown up, you'll spend the credits
+                    you have at the market to buy them. There are four handouts that detail all of the
+                    equipment in the game. You can look at them at any time, but you can only buy from
+                    them and put the equipment on your Mech during the Building phase.
+                </p>
+                <p class="rules">
+                    Most weapons and subsystems have unlimited stock, and will never sell out. The 
+                    number of such equipment you can equip on your Mech is limited only by the
+                    credits you have to spend on them and the space you have in your Mech. 
+                    <b>Unique Weapons and Subsystems</b> on the other hand require a card to unlock,
+                    and only have in stock the number that's in parentheses above their name on the
+                    unique subsystems page.
+                </p>
+            </div>
+            <div class="subsystem">
+            <p class="rules"><b>Selling:</b> 
+                If you have weapons or subsystems that you don't want to use anymore, you can 
+                sell them back to the market. Unless you have a card that says otherwise, when 
+                you sell a weapon or subsystem, you get 50% of the market price, round down. 
+            </p>
+            <p class="rules">
+                You cannot sell more than 4 weapons in a round. However, you can unequip weapons
```

Comment:
> You cannot sell more than 4 weapons in a round.

After some further thought, I think this should change back to the old rules: "You cannot sell more than 4 weapon space or 1 weapon that consume more than 4 weapon spaces, per round."
This should help dissuade players from drastically changing their builds in-between a round.

Reply:
Don't players already get penalized for hard switches from losing half the $ from selling the weapons? I'm concerned that we're double-penalizing players for switching, and breaking the promise of play/counter-play. What if someone bought a bunch of missiles or lasers, and their opponent has either totally or mostly nullified them with defenses? Should they lose not only the round where they found those defenses, but also be at a disadvantage the round after as well if they can't pivot?

Reply:
> Don't players already get penalized for hard switches from losing half the $ from selling the weapons?

Without this 2nd rule: round 3, you get 110 credits minimum (assuming you won the previous round), and assuming you sold all your weapons from previous rounds (toss in the 75% connections card), you can completely re-spec your mech into an entirely different weapon type.

> ...breaking the promise of play/counter-play

I want players to be able to think about and plan for, e.g., round 3 when they are on round 1. If you go wholesale into missiles for 2 rounds and get completely countered by round 3, they beat your strategy. If you plan ahead what you need to sell/remove to pivot into a different weapon system, your opponent will have a more complicated challenge. (I regularly mix weapon types to ensure I can't be fully countered.) It's typically damage-optimal to focus on one type, but you are putting all your eggs in once basket, as the saying goes.
Vs.
If you go wholesale into lasers for 2 rounds, then sell them all and re-spec to autocannons on round 3, then it's harder to counter and ruins some play-counterplay strategy because you would have to blindly purchase counters that aren't relevant to the target from your previous knowledge. Thus, purchasing counters will more-likely end up being a waste of money, and that's not much fun for the defender (imo). Plus, you still have the option of pulling weapons off your mech and re-spec'ing without selling if it's advantageous (just sell the weapons in future rounds). If the defender spends 15x4=60 credits on armor plates, to counter you, and you simultaneously sell _some_ weapons and re-spec, I think you'll be decently-matched in terms of cost.

Also, lasers and autocannons simply cannot be completely countered (even with 4 diffusive mesh + 1 chitin) because you can get the damage bonus from accuracy rolls to compensate (I added that clarification to the armor plating rules, for the next update). Lasers/Gauss get the bonus option of un-counterable raw damage via capacitors. Autocannons have ultra-dense slugs and, generally, have larger die sizes. _Note, I'm still working on the missile systems, we'll see if missiles can still be fully countered when I'm done. The thing about the current implementation is: a Guardian mech with a mix of SRM/MRM/LRMs (or a dedication to one type) can fire 104-138 missiles. That will do tremendous damage to a target with MIRV on plain terrain unless they have 2x E-pulse plating + Chitin plating + Infrared countermeasures + 2 LAMS, which achieves invulnerability (or a crazy, large quantity of LAMS, or etc.). That defense will cost you 150 credits + 5 ammo + 6 power + 2 cards, and leave you vulnerable to all other weapon types, and you still have to purchase standard Armor (the Guardian would have to spend ~236-294 credits in missile systems). 

Per our phone conversation yesterday and this previous paragraph, missiles were originally primary weapons, but I partially-forced them into an auxiliary role with an AMS buff (among other things) because getting 1-shot every round isn't much fun, imo. My solution might include a damage nerf to missiles (and appropriate cost reduction) to get around that problem._

With all of that laid out, what are your opinions/thoughts?

Reply:
whoa, that's a whole blog post! :D 
I think you've convinced me about the eggs-in-one-basket argument, so I'll make the change in this update. 
I think one-shot-kills (I colloquially call them "ganks") are fun for exactly one person, the killer, and they should be avoided. 

> "and leave you vulnerable to all other weapon types" 

well, not so much, since Chitin and IR CMs work for all weapon types.

Reply:
> whoa, that's a whole blog post! :D

I put in as much reasonable thought as I could, hahaha.


> > "and leave you vulnerable to all other weapon types"

>well, not so much, since Chitin and IR CMs work for all weapon types.

Touché, you would get universal plating benefit from Chitin and accuracy reduction from the IR CMs. I was tunnel-visioning the fact that you would have to use all 4 armor plating slots for e-pulse. "... and leave you more weak to builds focused on other weapon types" :D

Side note, I turned off my e-mail filters so now I'll be able to see your github updates :)